tools/docker/fabric-all-in-one/Dockerfile

```diff
@@ -71,14 +79,15 @@ RUN curl -sSL https://raw.githubusercontent.com/hyperledger/fabric/release-2.2/s
 RUN chmod +x bootstrap.sh
 # Run the bootstrap here so that at least we can pre-fetch the git clone and the binary downloads resulting in
 # faster container startup speed since these steps will not have to be done, only the docker image pulls.
-RUN /bootstrap.sh $FABRIC_VERSION $FABRIC_VERSION -d
+RUN /bootstrap.sh ${FABRIC_VERSION} 1.4.8 -d
```

Comment:
We can have a separate variable for the CA version the same way we have one for the Fabric version. No need to hardcode it.

Reply:
Incorporated the changes. Please have a re-look.